### PR TITLE
fix https://github.com/apache/skywalking/issues/4512

### PR DESCRIPTION
**Pull request by vision-ken**
Closes https://github.com/apache/skywalking/issues/4512

### CODE REVIEW

**Comment by innerpeacez** [comment]
Elasticsearch related configuration is expressed as `elasticsearch.config.xxx`, and you also need to modify the `deployment.yaml` file and give tips in the `values.yaml` file.
image @vision-ken

**Comment by innerpeacez** [comment]
Hello, do you want to continue to do this PR? @vision-ken

**Comment by vision-ken** [comment]
@innerpeacez 
Can't find  deployment.yaml file in the project.

**Comment by innerpeacez** [comment]
> @innerpeacez
> Can't find `deployment.yaml` file in the project.

`oap-deployment.yaml` and `es-init.job.yaml`

**Comment by vision-ken** [comment]
@innerpeacez 
I think it should be added here but not  `elasticsearch.config.xxx`:
```yaml
oap:
  env:
    SW_ES_USER: "elastic"
    SW_ES_PASSWORD: "xxx"
```

**Comment by vision-ken** [comment]
I'v changed the  `es-init.job.yaml` file  and `oap-deployment.yaml`  file no need to change.

**Comment by innerpeacez** [comment]
It doesn't matter where the configuration is placed, as long as it can be read when the container starts. I want to put it under the `elasticsearch.config.xx` related to put together for easy management.Because some other configurations of es have been placed under `elasticsearch.config.xx`

**Comment by vision-ken** [comment]
@innerpeacez Ok, I’ll make a change.

**Comment by vision-ken** [comment]
@innerpeacez 
Done.

File: chart/skywalking/templates/es-init.job.yaml [hunk 1]
```diff
@@ -55,7 +55,11 @@ spec:
   {{- else }}
           value: "{{ .Values.elasticsearch.config.host }}:{{ .Values.elasticsearch.config.port.http }}"
 {{- end }}
+        - name: SW_ES_USER
```
Comment (innerpeacez) [inline chart/skywalking/templates/es-init.job.yaml]: Need to determine whether these two keys exist, some users do not need to fill in these two keys. And you need to note that `elasticseach.config.xx` is currently only available to users who already have elasticseach.

File: chart/skywalking/values.yaml [hunk 2]
```diff
@@ -56,8 +56,6 @@ oap:
   env:
     # more env, please refer to https://hub.docker.com/r/apache/skywalking-oap-server
```
Comment (innerpeacez) [inline chart/skywalking/values.yaml]: `values-es6.yaml` also needs tips

**Pull request merged**